docs/source/getting_started.rst

```diff
@@ -4,42 +4,71 @@
 Getting Started
 ===============
 
-
----------------
-Prerequisites
----------------
+------------
+Dependencies
+------------
 
 PyNWB has the following minimum requirements, which must be installed before you can get started using PyNWB.
 
-#. Python 3.x
-
-   * Before moving forward, make sure `setuptools <https://pypi.python.org/pypi/setuptools>`_ is installed with your version of Python.
-
-#. HDF5
-#. MPI
-
+#. Python 2.7 or Python 3.6
+#. pip
 
 The NWB format provides a formal specification for storing neurophysiology data in HDF5. HDF5 provides support
 for parallel I/O using MPI-IO, and therefore requires MPI.
 
----------------
+------------
 Installation
----------------
+------------
+
+* Required Packages
+
+  #. numpy
+  #. h5py
+  #. ruamel.yaml
+  #. python-dateutil
+  #. six
+  #. requests
+
+Install from pypi
+-----------------
+
+To install or update PyNWB distribution from PyPI simply run:
+
+.. code::
+
+   $ pip install -U pynwb
+
+Install from Git repository (for development)
+---------------------------------------------
+
+For development an editable install is recommended.
+
+.. code::
+
+   $ mkvirtualenv pynwb
+   $ git clone git@github.com:NeurodataWithoutBorders/pynwb.git
+   $ cd pynwb
+   $ pip install -r requirements.txt
+   $ pip install -e .
+
+Install latest dev version
```

Comment:
Move this after `Install from pypi` and before `Install from Git repository ...`